pkg/serverless/trigger/events/events.go

```diff
@@ -228,9 +228,10 @@ type KinesisRecord struct {
 // EventBridgeEvent is used for unmarshalling a EventBridge event.  AWS Go
 // libraries do not provide this type of event for deserialization.
 type EventBridgeEvent struct {
-	DetailType string `json:"detail-type"`
-	Source     string
-	StartTime  string
+	DetailType string                 `json:"detail-type"`
+	Source     string                 `json:"source"`
+	Time       time.Time              `json:"time"` // Time only has precision to the nearest second
+	Detail     map[string]interface{} `json:"detail"`
```

Comment:
You shouldn't need to add the `json:...` for source, time, and detail. Since we're only ever unmarshalling the json, go will ignore the casing. See https://go.dev/play/p/qcrfqjCfZ7J